### DISCUSSION TITLE
'last' command equivalent for termux?

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-app#2534] [Q&A]
Was looking to implement a login counter and it seems 'last' is unavailable on Termux?
Is there any other way I could fetch login count information via available system files?

**Comment by agnostic-apollo** [comment]
Update `$PREFIX/bin/login` or similar file I guess.

**Reply by nathaneltitane** [reply]
Specifics?

**Reply by agnostic-apollo** [reply]
Isn't that for you to decide depending on your use case. You can add something like `date +"%Y-%m-%d %H.%M" > ~/last_login` in `login` script to store timestamp of login or increment a counter. Of course, termux runs `login` script for every session, I plan on adding a env variable that will only be true for first session so that users can use it in scripts as "real" `login`.
